### PR TITLE
Vendor all dependencies

### PR DESCRIPTION
**Pull request by kevinburke**
Updates mattn/goreman#73.

This work was sponsored by [Sourcegraph](https://sourcegraph.com)

### CODE REVIEW

**Comment by kevinburke** [comment]
So the tests are failing because of the global `wg`. I'll try to get a fix in for that soon, I think we need to attach it to an object.

**Comment by mattn** [comment]
I resolved issue of wg.

File: goreman_test.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ func startGoreman(ctx context.Context, t *testing.T, ch <-chan os.Signal, file [
 }
 
 func TestGoreman(t *testing.T) {
+	defer wg.Wait()
```
Comment (mattn) [inline goreman_test.go]: please remove this `defer wg.Wait()` at all.

Comment (kevinburke) [reply goreman_test.go]: Done!

**Comment by kevinburke** [comment]
@mattn thoughts on this change? It is a requirement in order to add goreman to Homebrew.

I am not super happy with Go modules and could replace with something like dep, if you want.

**Comment by mattn** [comment]
Thanks

**Pull request merged**